worker.tf

```diff
@@ -20,17 +20,7 @@ module "worker" {
 
   security_group_ids = [
     exoscale_security_group.all_machines.id,
-    exoscale_security_group.worker.id,
   ]
 
   bootstrap_bucket = var.bootstrap_bucket
 }
-
-resource "exoscale_domain_record" "router_member" {
-  count       = var.worker_state == "Running" ? var.worker_count : 0
-  domain      = exoscale_domain.cluster.id
-  name        = "router-member"
-  ttl         = 60
-  record_type = "A"
-  content     = module.worker.ip_address[count.index]
-}
```

Comment:
I guess this means it won't be possible to run the router on normal worker nodes?

Reply:
Yeah, it's not intended to run the router on regular workers in the default setup as far as I understand